modules/dnn/src/softmax.cpp

```diff
@@ -0,0 +1,32 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2022, Intel Corporation, all rights reserved.
+// Third party copyrights are property of their respective owners.
```

Comment:
Please remove these lines.

Reply:
Revert the last commit. As a result, it will be deleted.